frontend/src/views/players/players.tsx

```diff
@@ -32,30 +29,18 @@ const PlayersPage = () => {
 
 
 const PlayersList = withRouter(({history}) => {
-    const [player, setPlayers] = useState({loading: true, players: []});
     const {url} = useRouteMatch();
 
-    useEffect(() => {
-        async function load() {
-            const players = await playersApi.getAll();
-            setPlayers({loading: false, players});
-        }
-        load();
-    }, []);
-
     return (
         <Page title="Players">
             <nav>
                 <Link to={`${url}/create`}>Create player</Link>
             </nav>
 
             <div className="players">
-                {player.loading && <BusyOverlay/>}
-
                 <List itemRender={player => <PlayerItem {...player}/>}
-                      data={player.players}
-                      onItemClick={(i) => history.push(url + '/' + i.code)}
-                />
+                      data={playersApi.getAll()}
```

Comment:
you're fetching the data each time the component re-renders.

Reply:
keeping it fresh :D

Reply:
Not longer as fresh as it was but still tasty 🍕